pkg/api/v1/clients/memory/resource_client.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/solo-io/solo-kit/pkg/api/v1/clients"
 	"github.com/solo-io/solo-kit/pkg/api/v1/resources"
 	"github.com/solo-io/solo-kit/pkg/errors"
+	"github.com/solo-io/solo-kit/pkg/utils/kubeutils"
```

Comment:
This feels a bit awkward. Our solo-kit library attempts to separate out the persistence layers intentionally, and this seems to blend the kubernetes and in-memory implementations. Is there a way around this, or a side-effect of using types in our in-memory client that are kubernetes specific?

Reply:
Though I guess the below `labels` import already broke this paradigm, so we can ignore my earlier comment